lib/generators/wcc/model_generator.rb

```diff
@@ -1,14 +1,9 @@
 # frozen_string_literal: true
 
 module Wcc
-  class MenuGenerator < Rails::Generators::Base
+  class ModelGenerator < Rails::Generators::Base
     source_root File.expand_path('templates', __dir__)
-
-    def create_menus_migration
-      now = Time.now.strftime('%Y%m%d%H%M')
-      copy_file 'menu/generated_add_menus.ts',
-        "db/migrate/#{now}01_generated_add_menus.ts"
-    end
+    argument :model, type: :string
```

Comment:
Let's add a little validation to ensure that the 'model' argument is one of our known types for which we have a template